src/app/src/main/resources/application.properties

```diff
@@ -1,10 +1,11 @@
-spring.resources.add-mappings=true
+spring.resources.add-mappings=false
```

Comment:
This is the value that stops spring auto-handling all requests as resource requests and intercepting 404s